index.md

```diff
@@ -37,6 +37,30 @@ consumers of your library:
 A scope may be provided to a commit's type, to provide additional contextual information and
 is contained within parenthesis, e.g., `feat(parser): add ability to parse arrays`.
 
+## Examples
+
+### Commit message with body and breaking change
+```
+feat: allow provided config object to extend other configs
+
+BREAKING CHANGE: `extends` key in config file is now used for extending other config files
+```
+
+### Commit message with no body
+```
+docs: correct spelling of CHANGLOG
+```
+
+### Commit message with scope
+```
+feat(lang): added polish language
+```
+
+### Commit message for a fix using an (optional) issue number.
+```
+fix: issue #12 - minor typos in code
```

Comment:
This `issue #12` should be used in the footer and not in the commit message.

Reply:
So that means I have to also include a body? That's kind of annoying if there is otherwise no need for a body. I'll change this for now, but I think this rule should be reconsidered.